tox.ini

```diff
@@ -9,7 +9,18 @@ minversion = 1.7.0
 
 [testenv]
 deps =
-    -rtests/requirements.txt
+    attrs
+    pexpect >= 4.8.0
+    pyftpdlib >= 1.5.8
+    pytest
+    pytest-cov==4.0.0
+    pytest-xdist
+    uvloop; platform_system != "Windows"
+    sybil >= 1.3.0  # https://github.com/cjw296/sybil/issues/20#issuecomment-605433422
+    testfixtures
+    brotli; implementation_name != 'pypy'  # optional for HTTP compress downloader middleware tests
+    brotlicffi; implementation_name == 'pypy'  # optional for HTTP compress downloader middleware tests
```

Comment:
uvloop, brotli and brotlicffi should be moved into `extra-deps`.

Reply:
The same: I was just making tests, because when I did like you said, the CI tests was not approved